sdk/search/Azure.Search/src/Options/IndexDocumentsOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Search
+{
+    /// <summary>
+    /// Options for <see cref="SearchIndexClient.IndexDocumentsAsync(Models.IndexDocumentsBatch{Models.SearchDocument}, IndexDocumentsOptions, System.Threading.CancellationToken)"/>.
+    /// </summary>
+    public class IndexDocumentsOptions : SearchRequestOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether to throw an exception on
+        /// any individual failure in the batch of document write operations.
+        ///
+        /// Set this to true if you're not inspecting the results of the Index
```

Comment:
White space is elided in XML doc comments unless you use `<para>`. Maybe this should be in `<remarks>`, unless you want devs to see it in IntelliSense. Maybe use `<para>` then.

Reply:
Removed the whitespace - I think it's all fine to present together.